dev/TOC.md

```diff
@@ -347,12 +325,46 @@
     - [PD](reference/key-monitoring-metrics/pd-dashboard.md)
     - [TiKV](reference/key-monitoring-metrics/tikv-dashboard.md)
   - [Adopters](adopters.md)
++ TiDB in Kubernetes
+  - [About TiDB Operator](tidb-in-kubernetes/overview.md)
+  + Get Started
+    - [DinD](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-dind.md)
+    - [GKE](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-gke.md)
+    - [Minikube](tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-minikube.md)
+  + Deploy
+    - [Prerequisites for TiDB in Kubernetes](tidb-in-kubernetes/deploy/prerequisites.md)
+    - [TiDB in AWS EKS](tidb-in-kubernetes/deploy/aws-eks.md)
+    - [TiDB in GCP](how-to/tidb-in-kubernetes/gcp-gke.md)
+    - [TiDB in Alibaba Cloud](how-to/tidb-in-kubernetes/alibaba-cloud.md)
+    - [TiDB in General Kubernetes](tidb-in-kubernetes/deploy/general-kubernetes.md)
+    - [Deploy TiDB Operator](tidb-in-kubernetes/deploy/tidb-operator.md)
```

Comment:
I'd suggest changing the order:
- Prerequisites
- Deploy TiDB Operator
- TiDB in General Kubernetes
- Public clouds